opentelemetry-sdk/src/opentelemetry/sdk/_configuration/__init__.py

```diff
@@ -27,15 +27,22 @@
 from opentelemetry import trace
 from opentelemetry.environment_variables import (
     OTEL_LOGS_EXPORTER,
+    OTEL_METRICS_EXPORTER,
     OTEL_PYTHON_ID_GENERATOR,
     OTEL_TRACES_EXPORTER,
 )
+from opentelemetry.metrics import set_meter_provider
```

Comment:
In that case we should change the import of `trace`, importing should be minimized to only the necessary components.

Reply:
On this point, I'll note that _all_ the docs and examples do it like the existing code (`from opentelemetry import trace`), not like this. There are literally 0 occurrances of `from opentelemetry.trace import` in this repo (although I'm looking at a v1.11.0 checkout right now) that are importing `set_tracing_provider`.

The only use of `from opentelemetry.trace import` in the docs is the `set_span_in_context` example, which is 16 lines below `from opentelemetry import trace` in the same doc.

I personally don't like the style of importing _functions_ and prefer to import either classes or modules, so I prefer the currently-documented idiom.

Reply:
I think I have to clarify myself. Yes, so far we have not been checking how we import in our lint checkers. That is ok, sometimes it is necessary to import modules directly instead of "leaf-level" objects.

I _personally prefer_ to make the imports as specific as possible as this minimizes the amount of code:

```python
import a
...
a.function()
...
a.function()
```
```python
from a import function
...
function()
...
function()
```

That's two less instances of `a` in the code in the second example.

Now, this is my only argument, and it is not very important which approach we follow, since it barely has any consequences: this is a nit.

Reply:
BTW, I am ok with this conversation being marked as resolved by you or @srikanthccv in order not to block this PR from being merged.

Reply:
to each our own 👍 , i'm fine with either way.